src/exit/_Exit.rs

```diff
@@ -0,0 +1,8 @@
+use c_types::c_int;
+
+#[no_mangle]
+pub unsafe extern "C" fn _Exit(ec: c_int) -> ! {
+    syscall!(EXIT_GROUP, ec);
+    syscall!(EXIT, ec);
+    loop {}
```

Comment:
looks like I lost my original reply in the ether.

the loop on line 7 is unreachable, since the `EXIT` syscall doesn't return. The loop serves to make the function divergent: without the loop, the return type of the `syscall!` macro, `()`, conflicts with the specified return type of `!`, so `rustc` refuses to compile it. Adding the loop lets `rustc` compile without complaint.

The loop is similar in spirit to the loop in the original C. See reasoning here: http://git.musl-libc.org/cgit/musl/commit/src/exit/_Exit.c?id=0c05bd3a9c165cf2f0b9d6fa23a1f96532ddcdb3